### PR TITLE
Completed fetch-in-react exercise

### PR DESCRIPTION
**Pull request by brandon-moy**
image

### CODE REVIEW

File: fetch-in-react/src/app.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ isLoading: false });
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(res => res.json())
+      .then(data => this.setState({ users: data }))
+      // eslint-disable-next-line no-console
+      .catch(err => console.err(err));
```
Comment (thebearingedge) [inline fetch-in-react/src/app.jsx:19]: The method is console.error and the linter allows it

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: fetch-in-react/src/app.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ isLoading: false });
```
Comment (thebearingedge) [inline fetch-in-react/src/app.jsx:14]: Also, we should wait until we have the data before setting this to true

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: fetch-in-react/src/app.jsx (hunk 1 above)
Comment (brandon-moy) [reply fetch-in-react/src/app.jsx]: Oops, my mistake. This has been addressed. Working on the second comment now 
image

File: fetch-in-react/src/app.jsx (hunk 2 above)
Comment (brandon-moy) [reply fetch-in-react/src/app.jsx:14]: Makes sense. I update the state as the same time the user data is updated here.

image

image

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**